src/hearings/components/listing-information-summary/listing-information-summary.component.html

```diff
@@ -107,12 +101,6 @@ <h2 class="govuk-heading-m">Listing information summary</h2>
 
 <ng-template #listingContentMultiDay>
   <dl class="govuk-summary-list govuk-!-margin-bottom-0 govuk-!-padding-right-2">
-    <div class="govuk-summary-list__row" *ngIf="!isCaseStatusListed()">
```

Comment:
Adeel said this should be retained, we need him to correct the testing ticket.

Reply:
I have added Status back.
Do we still want to go ahead with the change 'party' to 'participant' as per the ticket? or should I revert this completely?